### PR TITLE
Add theme switching and set fluent theme as default

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: packages/devextreme-cli/src/templates/react/application/src/theme.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  return themes[themes.indexOf(theme) + 1] || themes[0];
+}
+
+function getCurrentTheme() {
+  return currentTheme;
+}
+
+function toggleTheme(prevTheme<%=#isTypeScript%>: string<%=/isTypeScript%>) {
+  const isCurrentThemeDark = prevTheme === 'dark';
+
```
Comment (dxvladislavvolkov) [inline packages/devextreme-cli/src/templates/react/application/src/theme.tsx:17]: Remove the unnecessary line

File: packages/devextreme-cli/src/templates/vue-v3/application/src/components/user-panel.vue [hunk 2]
```diff
@@ -1,114 +1,93 @@
...
+      v-if="menuMode !== 'list'"
+      stylingMode="text"
+      :icon="'https://js.devexpress.com/Demos/WidgetsGallery/JSDemos/images/employees/06.png'"
+      :showArrowIcon="false"
       :items="menuItems"
+      displayExpr="text"
+      keyExpr="id"
+      :elementAttr="{
+      class: 'user-button'
+    }"
```
Comment (dxvladislavvolkov) [inline packages/devextreme-cli/src/templates/vue-v3/application/src/components/user-panel.vue:13]: Please fix the alignment in the code

File: packages/devextreme-cli/src/templates/vue-v3/application/src/components/user-panel.vue [hunk 3]
```diff
@@ -1,114 +1,93 @@
...
+      :dropDownOptions="{
+      width: '150px'
+    }"
+    />
+    <dx-list v-if="menuMode === 'list'"
+             :items="menuItems"
```
Comment (dxvladislavvolkov) [inline packages/devextreme-cli/src/templates/vue-v3/application/src/components/user-panel.vue:19]: Please fix the alignment in the code

File: packages/devextreme-schematics/src/add-app-template/schema.json [hunk 4]
```diff
@@ -39,7 +39,7 @@
     "dxversion": {
       "type": "string",
       "description": "The DevExtreme version",
-      "default": "24.1.4"
+      "default": "~24.1.7"
```
Comment (dxvladislavvolkov) [inline packages/devextreme-schematics/src/add-app-template/schema.json:42]: remove ~

Comment (dxvladislavvolkov) [reply packages/devextreme-schematics/src/add-app-template/schema.json:42]: and looks like we need use beta version

File: packages/devextreme-schematics/src/add-layout/files/src/app/shared/services/theme.service.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+function getNextTheme(theme?: Theme) {
+  return (theme && themes[themes.indexOf(theme) + 1]) || themes[0];
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ThemeService {
+
```
Comment (dxvladislavvolkov) [inline packages/devextreme-schematics/src/add-layout/files/src/app/shared/services/theme.service.ts:17]: Remove the unnecessary line

File: packages/devextreme-schematics/src/utility/latest-versions.ts [hunk 6]
```diff
@@ -1,6 +1,6 @@
 // TODO: implement
 export const latestVersions = {
-  'devextreme': '24.1.4',
-  'devextreme-angular': '24.1.4',
+  'devextreme': '~24.1.7',
+  'devextreme-angular': '~24.1.7',
```
Comment (dxvladislavvolkov) [inline packages/devextreme-schematics/src/utility/latest-versions.ts:4]: Use unstable version of devextreme

File: templates-generator/react-config.js [hunk 7]
```diff
@@ -143,19 +143,21 @@ module.exports = {
...
                 'import \'./themes/generated/theme.base.css\';\n',
+                'import \'./themes/generated/theme.base.dark.css\';\n',
                 'import \'./themes/generated/theme.additional.css\';\n',
+                'import \'./themes/generated/theme.additional.dark.css\';\n',
             ]
         },
         {
             glob: 'src/app-routes.tsx',
             definitions: [
-                    ', NewPagePage'
```
Comment (dxvladislavvolkov) [inline templates-generator/react-config.js:152]: Why did you change it?

Comment (GoodDayForSurf) [reply templates-generator/react-config.js:152]: hmm.. in my IDE this place not changed

**Review by dxvladislavvolkov — APPROVED** [review]
